From 4a08c9e07fa8b7a1d6a76b927ea97e3eaa784845 Mon Sep 17 00:00:00 2001
From: Ezequiel Garcia <ezequiel@vanguardiasur.com.ar>
Date: Sun, 26 Oct 2014 16:16:25 -0300
Subject: [PATCH] compile: Remove -Werror and avoid build failures because of
 warnings

Using -Werror on production is a bad idea, as otherwise harmless warnings
get treated as errors breaking builds. This is currently the case with
Nios-II architecture toolchain, warning about a deprecated macro:

../usr/nios2-buildroot-linux-gnu/sysroot/usr/include/features.h:148:3:
error: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Werror=cpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"

Instead of trying to fix the macro, let's fix the real issue here,
and remove -Werror.

Signed-off-by: Ezequiel Garcia <ezequiel@vanguardiasur.com.ar>
---
 src/makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/makefile b/src/makefile
index 294dc08..4ac4c43 100644
--- a/src/makefile
+++ b/src/makefile
@@ -32,7 +32,7 @@ ZLOG_MINOR=2
 # Fallback to gcc when $CC is not in $PATH.
 CC:=$(shell sh -c 'type $(CC) >/dev/null 2>/dev/null && echo $(CC) || echo gcc')
 OPTIMIZATION?=-O2
-WARNINGS=-Wall -Werror -Wstrict-prototypes -fwrapv
+WARNINGS=-Wall -Wstrict-prototypes -fwrapv
 DEBUG?= -g -ggdb
 REAL_CFLAGS=$(OPTIMIZATION) -fPIC -pthread $(CFLAGS) $(WARNINGS) $(DEBUG)
 REAL_LDFLAGS=$(LDFLAGS) -pthread
-- 
2.1.0

